cclib/parser/turbomoleparser.py

```diff
@@ -208,6 +228,53 @@ def extract(self, inputfile, line):
             self.set_attribute('vibirs', vibirs)
             self.set_attribute('vibdisps', vibdisps)
 
+        # In this section we are parsing mocoeffs and moenergies from
+        # the files like: mos, alpha and beta.
+        # $scfmo    scfconv=6   format(4d20.14)
+        # # SCF total energy is     -382.3457535740 a.u.
+        # #
+        #      1  a      eigenvalue=-.97461484059799D+01   nsaos=60
+        # 0.69876828353937D+000.32405121159405D-010.87670894913921D-03-.85232349313288D-07
+        # 0.19361534257922D-04-.23841194890166D-01-.81711001390807D-020.13626356942047D-02
+        # ...
+        # ...
+        # $end
+        if (line.startswith('$scfmo') or line.startswith('$uhfmo')) and line.find('scfconv') > 0:
+            if line.strip().startswith('$uhfmo_alpha'):
+                self.unrestricted = True
+
+            # Need to skip the first line to start with lines starting with '#'.
+            line = next(inputfile)
+            while line.strip().startswith('#') and not line.find('eigenvalue') > 0:
+                line = next(inputfile)
+
+            moenergies = []
+            mocoeffs = []
+
+            while not line.strip().startswith('$'):
+                info = line.split()
+                eigenvalue = float(info[2][11:].replace('D', 'E'))
+                orbital_energy = utils.convertor(eigenvalue, 'hartree', 'eV')
+                moenergies.append(orbital_energy)
+                single_coeffs = []
+                nsaos = int(info[3][6:])
+
+                while(len(single_coeffs) < nsaos):
+                    line = next(inputfile)
+                    single_coeffs.extend(Turbomole.split_molines(line))

```

Comment:
Can't you just use `split_molines` without `Turbomol`?

Reply:
So... then maybe it should go into `utils.py`?